### DISCUSSION TITLE
x轴与y轴的最大刻度值无法显示

### DISCUSSION THREAD

**Discussion by Shuao058** [GSI-CS-CO/chart-fx#634] [Q&A]
![微信截图_20231102102126](https://github.com/fair-acc/chart-fx/assets/98296380/041eafaa-837e-4138-9803-dc9cd9f79b77)

![微信图片_20231102102301](https://github.com/fair-acc/chart-fx/assets/98296380/ffececbf-d970-4180-99f5-6d41ea451848)

x轴与y轴的最大刻度值被覆盖无法显示

**Comment by ennerf** [comment]
对不起，由于我们不支持中文服务，此问题将关闭。
